index.html

```diff
@@ -179,12 +190,13 @@ <h5>Warframe Drop Data</h5>
         let info = JSON.parse(localStorage.getItem("_wfinfo"))
         let date = new Date(info.timestamp)
 
-        $("#lastbuildtime").text(date.toISOString())
+        $("#lastbuildtime").text(date.toLocaleString())
 
-        let matches = window.location.hash.match(/\#?\/search\/(.*)\//)
+        let matches = window.location.hash.match(/\#?\/search\/(.*)\/(items|locations|both)/)
```

Comment:
This should also accept not having "items", "locations" or "both" (maybe both if nothing?), because of A) old links and B) open search

Reply:
All done in 2192f5b70930412db010d168ecd02a44ada5f939!